content/docs/fun/reputation.md

```diff
@@ -57,6 +57,21 @@ give/remove points._
 Users with this role cannot participate in the point system which they can be given/taken away points from. This also
 overrides the _Required role to receive/being taken away points from_.
 
+## Reputation roles
```

Comment:
Title Case maybe? Though the header on the control panel isn't, so I'm not sure.

Reply:
yea, I referenced the page specifically for this one and figured if i made it different I'd get a comment about not matching, but I agree, it felt wrong here.

Reply:
let's keep it as-is